packages/suite/src/components/wallet/WalletLayout/components/WalletLayoutNavigation.tsx

```diff
@@ -33,22 +33,45 @@ const NavLink = styled.div<{ active?: boolean }>`
 
 const NavLinkText = styled.div`
     position: relative;
+    z-index: 1;
+`;
+
+const Badge = styled.span`
+    position: absolute;
+    top: -14px;
+    right: -27px;
+    padding: 2px 4px 0 4px;
+    border-radius: 4px;
+    background: ${props => props.theme.BG_LIGHT_GREEN};
+    color: ${props => props.theme.TYPE_GREEN};
+    font-size: 12px;
```

Comment:
using constants is preferred:

```suggestion
    font-size: ${variables.FONT_SIZE.TINY};
```

Reply:
Fixed.